app/models/user_mailbox.rb

```diff
@@ -0,0 +1,44 @@
+class UserMailbox
+  attr_reader :user
+
+  def initialize(user)
+    raise ArgumentError.new("Must provide a user") if user == nil
```

Comment:
Since the three of you seem to be against the guard I went ahead and removed it. 

@mjgiarlo the `user` parameter will come from the controller which is why I was trying to make sure it wasn't `nil`. As a developer I would rather see an error that indicates that "no user was passed" than "nil didn't have property xyz" (and then realize oh... the user must have been nil)